provision/acc_provision/versions.yaml

```diff
@@ -45,6 +45,7 @@ versions:
     opflex_server_version: 5.1.1.0.1ae238a
     gbp_version: 5.1.1.0.1ae238a
   5.2:
+    acc_provision_operator_version: newacioperator
```

Comment:
this has probably crept in from the dev env, it should be the same as others below...